roles/container-engine/containerd/defaults/main.yml

```diff
@@ -76,3 +76,6 @@ containerd_limit_mem_lock: "infinity"
 
 # If enabled it will use config_path and disable use mirrors config
 containerd_use_config_path: false
+
+# If enabled it will allow kubespray to attempt setup even if the distribution is not supported, for unsupported distributions this can lead to a failed setup in some cases
```

Comment:
```suggestion
# If enabled it will allow kubespray to attempt setup even if the distribution is not supported. For unsupported distributions this can lead to unexpected failures in some cases
```